src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DynamicFiles/RazorDocumentExcerptService.cs

```diff
@@ -87,7 +89,7 @@ internal class RazorDocumentExcerptService(
         var remainingSpan = excerptSpan;
         foreach (var span in sorted)
         {
-            if (excerptSpan.Length > 0)
+            if (excerptSpan.Length == 0)
```

Comment:
Intentional?

Reply:
Yes. Previously the loop was allowed to continue if `i < sorted.Count && excertSpan.Length > 0`. Using the `excerptSpan.Length` check to exit the loop early means that it has to be reversed.

Reply:
FWIW, I think you saw this by going commit-by-commit, right? In this particular case, I had originally blindly moved `excerptSpan.Length > 0` into the loop in an early commit. When unit tests failed, I fixed it.

Reply:
Indeed, I did see this going commit-by-commit, and completely missed or forgot, the change to the loop.